SteamKit2/SteamKit2/Util/VZipDeltaUtil.cs

```diff
@@ -16,52 +16,49 @@ class VZipDeltaUtil
 
         public static byte[] Decompress( byte[] buffer, byte[] sourceChunkData )
         {
-            using ( MemoryStream ms = new MemoryStream( buffer ) )
-            using ( BinaryReader reader = new BinaryReader( ms ) )
+            using MemoryStream ms = new MemoryStream( buffer );
+            using BinaryReader reader = new BinaryReader( ms );
+            if ( reader.ReadUInt16() != VZipHeader )
             {
-                if ( reader.ReadUInt16() != VZipHeader )
-                {
-                    throw new Exception( "Expecting VZipHeader at start of stream" );
-                }
-
-                if ( reader.ReadChar() != Version )
-                {
-                    throw new Exception( "Expecting VZip version 'd'" );
-                }
+                throw new Exception( "Expecting VZipHeader at start of stream" );
+            }
 
-                // This is also the CRC of the chunk
-                /* uint secondaryCRC = */ reader.ReadUInt32();
+            if ( reader.ReadChar() != Version )
+            {
+                throw new Exception( "Expecting VZip version 'd'" );
+            }
 
-                byte[] properties = reader.ReadBytes( 5 );
-                byte[] deltaBuffer = reader.ReadBytes( ( int )ms.Length - HeaderLength - FooterLength - 5 );
+            // This is also the CRC of the chunk
+            /* uint secondaryCRC = */
+            reader.ReadUInt32();
```

Comment:
newline here seems wrong compared to old code

Reply:
Autoformatting 😠